src/framework/entity.js

```diff
@@ -465,11 +468,11 @@ class Entity extends GraphNode {
 
         if (enableFirst) {
             // do not cache the length here, as enableList may be added to during loop
-            for (let i = 0; i < this._app._enableList.length; i++) {
-                this._app._enableList[i]._onHierarchyStatePostChanged();
+            for (let i = 0; i < _enableList.length; i++) {
+                _enableList[i]._onHierarchyStatePostChanged();
```

Comment:
Presumably this won't work in the multi-application case?

Reply:
I thought about it, and I think it should? This is only used inside a single function, those apps don't run in parallel, but in serial.